test/integration/stats_integration_test.cc

```diff
@@ -214,9 +205,10 @@ TEST_P(ClusterMemoryTestRunner, MemoryLargeClusterSizeWithStats) {
   // 2019/04/27  6733     50213               Use SymbolTable API for HTTP codes
   // 2019/05/31  6866     50157               libstdc++ upgrade in CI
   // 2019/06/03  7199     49393               absl update
-  // 2019/06/06  7208     49650               make memory targets approximate
+  // 2019/06/03  7243     49412       49700   macros for exact/upper-bound memory checks
```

Comment:
Don't rewrite history :) add a new line.

Reply:
fixed; I think I made a merge error; it was not my intent to rewrite history :)